src/spikeinterface/generation/tests/test_hybrid_tools.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import numpy as np
+from pathlib import Path
+
+from spikeinterface.core.generate import generate_ground_truth_recording
+from spikeinterface.core import Templates, BaseRecording
+from spikeinterface.preprocessing.motion import correct_motion, load_motion_info
+from spikeinterface.generation.hybrid_tools import estimate_templates_from_recording, generate_hybrid_recording
+
+
+if hasattr(pytest, "global_test_folder"):
+    cache_folder = pytest.global_test_folder / "generation"
+else:
+    cache_folder = Path("cache_folder") / "generation"
+
+
+def test_generate_hybrid_no_motion():
+    rec, sorting = generate_ground_truth_recording(sampling_frequency=20000)
+    hybrid, sorting = generate_hybrid_recording(rec)
```

Comment:
@yger can you add also a test for passing an external `sorting` and external `templates`?

For testing everything with and without drift, you can make a fixture to create the recording and estimate motion and use parameterized to either set `folder=None/motion_folder` :)

Reply:
Yes I am agree. We should have stronger tests.

Reply:
we should use the generate_drifting function everywhere in theses tests instead of the core one.